redact/redact_requests.py

```diff
@@ -58,23 +58,18 @@ def __init__(
         api_key: Optional[str] = None,
         httpx_client: Optional[httpx.Client] = None,
     ):
-
         self.redact_url = normalize_url(redact_url)
         self.api_key = api_key
         self.subscription_id = subscription_id
         self._headers = {"Accept": "*/*"}
-        self.retry_total_time_limit: float = 600  # 10 minutes in seconds
+        self.retry_total_time_limit: float = 600
 
-        if api_key:
+        if self.api_key:
             self._headers["api-key"] = self.api_key
-        if subscription_id:
+        if self.subscription_id:
             self._headers["Subscription-Id"] = self.subscription_id
 
-        # httpx.Client client is thread safe, see https://github.com/encode/httpx/discussions/1633
```

Comment:
Why remove comment? This info took some time to get

Reply:
Sorry I think it was an accident when merging. I will add it back in.

Reply:
done